examples/ftd_api_scripts/timerange.py

```diff
@@ -0,0 +1,104 @@
+from ftd_api_resources.access_token import get_access_token
+
+def main():
+    """
+    End to end example of code that updates an intrusion rule.
+    Requires Python v3.0 or greater and the reqeusts library.
+    You must update the values for host, port, user, and password to connect to your device.
+    """
+    host = 'u32c01p10-vrouter.cisco.com'
+    port = '20248'
+    user = 'admin'
+    passwd = 'Cisco@123'
+    access_token = get_access_token(host, port, user, passwd)
+    if not access_token:
+        print("Unable to obtain an access token. Did you remember to set host, port, user, and password?")
+        return
+    time_range = get_all_time_range(host, port, access_token)
+    if not time_range:
+        print('Unable to get time range')
+        return
+    tro = {
+    "name": "TRO_Python_Request_Test_2",
+    "description": "creating Time Range",
+    "effectiveStartDateTime": "2020-07-24T12:08",
+    "effectiveEndDateTime": "2020-07-28T12:08",
+    "recurrenceList": [
+    {
+      "recurrenceType": "DAILY_INTERVAL",
+      "days": ["MON","TUE","FRI"],
+      "dailyStartTime": "08:00",
+      "dailyEndTime": "09:00",
+      "type" : "recurrence"
+    },
+    {
+      "recurrenceType": "RANGE",
+      "rangeStartTime": "09:00",
+      "rangeEndTime": "11:00",
+      "rangeStartDay": "MON",
+      "rangeEndDay": "TUE",
+      "type" : "recurrence"
+    }
+  ],
+  "type": "timerangeobject"
+  }
+    time_range = post_time_range(host, port, access_token, tro)
+    if not time_range:
+        print('Unable to create time range object')
+        return
+    tro_update = {
+    'version': time_range['version'],
+    "name": "TRO_Python_Request_Test_Name_updated_2",
+    "description": "editing Time Range",
+    "effectiveStartDateTime": "2019-11-24T12:08",
+    "effectiveEndDateTime": "2019-12-28T12:08",
+    "recurrenceList": [
+    {
+      "recurrenceType": "DAILY_INTERVAL",
+      "days": ["MON","TUE","FRI"],
+      "dailyStartTime": "08:00",
+      "dailyEndTime": "09:00",
+      "type" : "recurrence"
+    },
+    {
+      "recurrenceType": "RANGE",
+      "rangeStartTime": "09:00",
+      "rangeEndTime": "11:00",
+      "rangeStartDay": "MON",
+      "rangeEndDay": "TUE",
+      "type" : "recurrence"
+    }
+  ],
+  "type": "timerangeobject"
+    }
+    updated_tro = update_time_range(host, port, access_token, time_range['id'], tro_update)
+    if not updated_tro:
+        print('Unable to update time range object')
+        return
+    tro = get_time_range(host, port, access_token, updated_tro['id'])
+    if not tro:
+        print('Unable to get time range object')
+        return
+    print('TRO name is {}'.format(tro['name']))
+    returnCode = delete_time_range(host, port, access_token, tro['id'])
+    if not returnCode:
+        print('Unable to delete time range object')
+        return
+
+if __name__ == '__main__':
```

Comment:
Please accept parameters from the command line, and pass them to the function. Return 0 or 1 depending on results. For example, see the deploy code.

Reply:
Done